python/lsst/analysis/tools/interfaces/_actions.py

```diff
@@ -29,6 +29,7 @@
     "MetricResultType",
     "MetricAction",
     "PlotResultType",
+    "PlotElement",
```

Comment:
Did you intend to add this (and associated class below) on this ticket?

Reply:
No, when I switched branches it looks like my editor thought it was code still active in the buffer or something